pkgs/applications/science/biology/delly/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit, htslib, zlib, bzip2, lzma, ncurses, boost }:
+
+stdenv.mkDerivation rec {
+
+  name = "${pname}-${version}";
```

Comment:
Remove this line, this is already done inside the `mkDerivation`.

Reply:
I removed the `name` attribute, is this the reason why the build failed?